testsuite/MDAnalysisTests/test_selections.py

```diff
@@ -61,6 +61,9 @@ def _write_selection(self, **kwargs):
         g.write_selection(self.namedfile, **kwargs)
         return g
 
+    def test_write_bad_mode(self):
```

Comment:
@jbarnoud,  `a+` is a valid mode for [open](https://docs.python.org/2/library/functions.html#open) but not for these writers. The tests check both that the proper exceptions are raised and, I guess, that we haven't implemented something new. Although I see your point , I am quite fine with testing for `a+` and moving on ;-).